project_closing/__manifest__.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2013 - 2015 Camptocamp
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    "name": "Project closing",
+    "version": "11.0.1.0.0",
+    "author": "Camptocamp, Métal Sartigan, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "website": "http://www.camptocamp.com",
+    "category": "Project Management",
```

Comment:
This should be "category": "Project" in order to agree with the other Odoo modules